aspnetcore/tutorials/first-mvc-app/new-field.md

```diff
@@ -58,13 +58,13 @@ Edit the */Views/Movies/Index.cshtml* file and add a `Rating` field:
 
 ::: moniker range=">= aspnetcore-3.0 < aspnetcore-5.0"
 
-[!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Views/Movies/IndexGenreRating.cshtml?highlight=16,38&range=24-63)]
+[!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/SnapshotNewField3/Views/Movies/IndexGenreRating.cshtml?highlight=16,38&range=24-63)]
 
 ::: moniker-end
 
 ::: moniker range=">= aspnetcore-5.0"
 
-[!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie5/Views/Movies/IndexGenreRating.cshtml?highlight=16,38&range=24-63)]
+[!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/SnapshotNewField5/Views/Movies/IndexGenreRating.cshtml?highlight=16,38&range=24-63)]
```

Comment:
Has this file changed between 3.1 and 5? I don't think the scaffolder changed. If that's true, you don't need a moniker here.